### PR TITLE
Minhquang/loginscreen

### PR DESCRIPTION
**Pull request by MQuang1609**
image

### CODE REVIEW

File: src/components/Login/index.tsx [hunk 1]
```diff
@@ -1,6 +1,34 @@
 import React from 'react';
-
+import { Button, Form, Input } from 'antd'
+import '../Login/login.scss'
+import logo from '../../assets/image/Rectangle 337.png'
+import loginImage from '../../assets/image/Artboard-113_WF-Automation-illustrations 1 (1).png'
 export const Login = () => {
   console.log('login page');
-  return <div>Login page</div>;
+  return <div>
```
Comment (ntc2019) [inline src/components/Login/index.tsx]: Cai div nay khong can thiet.

File: src/components/Login/login.scss [hunk 2]
```diff
@@ -70,31 +60,25 @@
...
-
     width: 8.2rem;
     height: 11.3rem;
 }
 
 .logo-img {
     width: 7.2rem;
     height: 7.2rem;
-
     border-radius: 1.16468rem;
```
Comment (ntc2019) [inline src/components/Login/login.scss]: Cái border radius nay no lẻ quá.

File: src/components/Login/index.tsx (hunk 1 above)
Comment (MQuang1609) [reply src/components/Login/index.tsx]: fixed done

File: src/components/Login/login.scss (hunk 2 above)
Comment (MQuang1609) [reply src/components/Login/login.scss]: fixed done

File: src/components/Login/index.tsx [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+          <Form layout="vertical" >
+            <div className='login__input'>
+              <label className='login__input--label'>Username</label>
+              <Input placeholder='Enter your username'></Input>
+            </div>
+            <div className='login__input'>
+              <label className='login__input--label'>Password</label>
+              <Input type='password' placeholder='Enter your password'></Input>
+            </div>
+            <Button className='login__btn'>Login</Button>
```
Comment (Khanhnqps10920) [inline src/components/Login/index.tsx]: please use double quotes for JSX attribute, single quote for Javascript code.

File: src/components/Login/login.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            box-sizing: border-box;
+            /* Auto layout */
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            padding: 0rem 3.2rem;
+            width: 50rem;
+            height: 45.6rem;
+            border-left: 0.1rem solid rgba(26, 30, 41, 0.12);
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]: create a scss variable

File: src/components/Login/login.scss [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+                .logo__img {
+                    width: 7.2rem;
+                    height: 7.2rem;
+                    border-radius: 1.2rem;
+                }
+
+                .logo__text {
+                    width: 8.2rem;
+                    height: 2.9rem;
+                    font-family: 'Gilroy';
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss:63]: setup font family at global file scss

File: src/components/Login/login.scss [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+                align-items: flex-start;
+                padding: 0rem;
+                width: 32.8rem;
+                height: 5.8rem;
+
+                .login__input--label {
+                    width: 32.8rem;
+                    height: 1.2rem;
+                    margin: 1rem 0;
+                    font-family: 'Poppins';
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]: setup font family at global file

File: src/components/Login/login.scss [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                    box-sizing: border-box;
+                    /* Auto layout */
+                    display: flex;
+                    flex-direction: row;
+                    align-items: center;
+                    padding: 0.8rem 1.6rem;
+                    width: 32.8rem;
+                    height: 4rem;
+                    background: #FFFFFF;
+                    border: 0.1rem solid #D6E4EC;
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]: scss var for color

File: src/components/Login/login.scss [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+                    font-family: 'Poppins';
+                    font-style: normal;
+                    font-weight: 500;
+                    font-size: 1.2rem;
+                    line-height: 1.8rem;
+                    display: flex;
+                    align-items: center;
+                    letter-spacing: 0.05rem;
+                    /* Primary / Dark * 500 */
+                    color: #343C44;
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]: scss var

File: src/components/Login/index.tsx [hunk 9]
```diff
@@ -1,6 +1,34 @@
 import React from 'react';
-
+import '../../../src/index.scss'
+import { Button, Form, Input } from 'antd'
+import '../Login/login.scss'
+import logo from '../../assets/image/Rectangle 337.png'
+import loginImage from '../../assets/image/Artboard-113_WF-Automation-illustrations 1 (1).png'
```
Comment (Khanhnqps10920) [inline src/components/Login/index.tsx]: use public directory for this image

File: src/components/Login/login.scss [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+.login {
+    justify-content: center;
+    align-items: center;
+
+    .login__wrapper {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
    &__wrapper {
```

File: src/components/Login/login.scss [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+        flex-direction: row;
+        align-items: center;
+        padding: 7.2rem 0rem;
+        width: 100rem;
+        height: 60rem;
+        background: #FFFFFF;
+        /* Colors/Primary White */
+        border-radius: 1.6rem;
+
+        .login__wrapper--left {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
        &--left
```

File: src/components/Login/login.scss [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            align-items: center;
+            width: 50rem;
+            height: 45.6rem;
+        }
+
+        .login__wrapper--right {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
        &--right {
```

File: src/components/Login/login.scss [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+            .logo {
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+                gap: 1.2rem;
+                width: 8.2rem;
+                height: 11.3rem;
+
+                .logo__img {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
               &__img {
```

File: src/components/Login/login.scss [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+                    width: 7.2rem;
+                    height: 7.2rem;
+                    border-radius: 1.2rem;
+                }
+
+                .logo__text {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
                &__text {
```

File: src/components/Login/login.scss [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+                    font-size: 2.4rem;
+                    line-height: 2.9rem;
+                    /* identical to box height */
+                    letter-spacing: 0.016em;
+                    /* Stroke 2nd */
+                    color: #1A1E29;
+                }
+            }
+
+            .login__input {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
            &__input {
```

File: src/components/Login/login.scss [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+                display: flex;
+                margin-bottom: 2rem;
+                flex-direction: column;
+                align-items: flex-start;
+                padding: 0rem;
+                width: 32.8rem;
+                height: 5.8rem;
+
+                .login__input--label {
```
Comment (Khanhnqps10920) [inline src/components/Login/login.scss]:
```suggestion
                &--label {
```

**Comment by Khanhnqps10920** [comment]
đổi tên file hình cho hợp lý nha em @MQuang1609

**Pull request merged**